requirements.in

```diff
@@ -17,7 +17,7 @@ psycopg2==2.7.3.2
 python-magic==0.4.15
 pysolr==3.6.0
 Pylons==0.9.7
-python-dateutil>=1.5.0,<2.0.0
+python-dateutil>=1.5.0
```

Comment:
@davidread any reason not to pin this to 2.7.3?

Reply:
It's been our policy to run `pip-compile --output-file requirements.txt requirements.in --upgrade` (I think --upgrade is new), which upgrades any indirect dep versions in requirements.txt that have looser version specifications. By unpinning dateutil in requirements.in I'm simply getting it included in that batch of upgrades.

I wouldn't put main deps (like flask) in this category - because there is a lot of potential impact and we should put more thought into doing manual upgrades. But I think for smaller libraries like dateutil, putting in some automation to update them makes sense.

We guard against any breakage by only running this command early in the dev cycle for each release, so that by release time any little problems are fixed.

And of course it's generally better to keep up with newer versions of things, because they are more maintained and have known vulnerability fixes. It's easy to get behind for these utility libraries.